2023/11/26 17:44:48 [error] 289#289: *243 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET / HTTP/1.1", upstream: "http://172.16.238.200:8800/", host: "front.api.aftertime.prod"
2023/11/26 17:44:49 [error] 289#289: *243 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.16.238.200:8800/favicon.ico", host: "front.api.aftertime.prod", referrer: "http://front.api.aftertime.prod/"
2023/11/26 19:32:26 [error] 341#341: *886 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET / HTTP/1.1", upstream: "http://172.16.238.200:8800/", host: "front.api.aftertime.prod"
2023/11/26 19:32:26 [error] 341#341: *886 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.16.238.200:8800/favicon.ico", host: "front.api.aftertime.prod", referrer: "http://front.api.aftertime.prod/"
2023/11/26 19:32:39 [error] 341#341: *886 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET / HTTP/1.1", upstream: "http://172.16.238.200:8800/", host: "front.api.aftertime.prod", referrer: "http://localhost:81/"
2023/11/26 19:32:39 [error] 341#341: *886 connect() failed (111: Connection refused) while connecting to upstream, client: 172.21.0.1, server: front.api.aftertime.prod, request: "GET /favicon.ico HTTP/1.1", upstream: "http://172.16.238.200:8800/favicon.ico", host: "front.api.aftertime.prod", referrer: "http://front.api.aftertime.prod/"
